Makefile

```diff
@@ -0,0 +1,34 @@
+.PHONY: help clean test coverage release build
```

Comment:
Should there be documentation in this PR about using the Makefile locally?

Reply:
Yup, good call